sdk/storage/azure-storage-blob/azure/storage/blob/container_client.py

```diff
@@ -21,7 +21,7 @@
 
 import six
 
-from ._shared.shared_access_signature import BlobSharedAccessSignature
+from azure.storage.blob.shared_access_signature import BlobSharedAccessSignature
```

Comment:
Minor: Just a small stylistic tweak - could we move this to the bottom and use the form:
```python
from .shared_access_signature import BlobSharedAccessSignature
```

Reply:
Thanks for the suggestion! I've fixed it